src/server/server.ts

```diff
@@ -175,6 +181,33 @@ export class Server {
       next();
     });
 
+    // Add the options to the prometheus middleware most option are for http_request_duration_seconds histogram metric
+    const metricsMiddleware = promBundle({
+      includeMethod: true,
+      includePath: true,
+      includeStatusCode: true,
+      includeUp: true,
+      promClient: {
+        collectDefaultMetrics: {
+        }
+      }
+    });
+    // add the prometheus middleware to all routes
+    this.app.use(metricsMiddleware);
+
+    const counter = new client.Counter({
+      name: 'event_count',
```

Comment:
Probably good to add a verification service related prefix so it's easy to differentiate in grafana

Reply:
I agree on this request, is also possible to add a prefix using the prometheus configuration, but I agree that having a Const defined a the project level to name all other app related metrics is the recommended way, so is easier to identify the metrics regardless of the prometheus scrapping config that is being used.

Reply:
Done. Also updated the PR description including this change.